core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1525,24 +1541,20 @@ public static boolean isClassProperty(String key) {
   }
 
   static {
+
     // Precomputing information here avoids :
     // * Computing it each time a method is called
     // * Using synch to compute the first time a method is called
-    propertiesByKey = new HashMap<>();
-    validPrefixes = new HashSet<>();
-    validProperties = new HashSet<>();
 
     for (Property p : Property.values()) {
+      propertiesByKey.put(p.getKey(), p);
       if (p.getType().equals(PropertyType.PREFIX)) {
         validPrefixes.add(p.getKey());
       } else {
         validProperties.add(p.getKey());
       }
-      propertiesByKey.put(p.getKey(), p);
-    }
-
-    validTableProperties = new HashSet<>();
-    for (Property p : Property.values()) {
+      // exclude prefix types (avoids being able to set things like table.custom or
+      // table.constraint)
```

Comment:
Minor wording clarification (especially using "prevents" rather than "avoids" to make it clear it's important for blocking the behavior we don't want, and why).
```suggestion
      // exclude prefix types (prevents setting a prefix type, like table.custom or
      // table.constraint, directly, since they aren't valid properties on their own)
```